### ISSUE TITLE
HerA is missing from srclist_pumav3_EoR0LoBES_EoR1pietro_CenA-GP_2023-11-07.fits

### ISSUE THREAD

**Issue by d3v-null** [JLBLine/srclists#9]
image on the left is real data, calibrated from [GGSM.fits](https://github.com/GLEAM-X/GLEAM-X-pipeline/blob/master/models/GGSM_updated.fits) 

image on the right is model visibilities made with

```bash
hyperdrive vis-sim \
        --time-res 8 \
        --freq-res 80 \
        --metafits /data/complete/1409998752/20240910-190201/1409998752_metafits.fits \
        --output-model-files 1409998752_jlb_99src_model.ms \
        --num-sources 99 \
        --source-list /home/mwa/srclists/srclist_pumav3_EoR0LoBES_EoR1pietro_CenA-GP_2023-11-07.fits

docker run --rm -it -v${PWD}:${PWD} -w$PWD mwatelescope/mwa-demo:latest wsclean \
        -name "1409998752_jlb_99src_model" \
        -size 2048 2048 \
        -scale 30asec \
        -pol i \
        -niter 100 \
        -multiscale \
        -weight briggs 0 \
        -mgain 0.85 -gain 0.1 \
        -auto-threshold 1 -auto-mask 3 \
        -make-psf \
        -circular-beam \
        1409998752_jlb_99src_model.ms
```

![1409998752_GGSM_99src-image fits-1409998752_jlb_99src_model-image fits-image-2024-09-11-17-24-35](https://github.com/user-attachments/assets/20c6a9d1-c215-4164-8486-07638df731b7)

Would you accept a pull request to update this fits file with HerA from GGSM?

Thanks!

**Comment by JLBLine** [comment]
Yeah nothing wrong with chucking in HerA. I know version control means we can just add HerA to this exact FITS, and roll back to previous versions if we want, but only the path to the sky model gets saved during the COMMENTS in a `hyperdrive` run. So I would suggest making a new file with a different name, a.k.a `srclist_pumav3_EoR0LoBES_EoR1pietro_CenA-GP_2024-09-12.fits`. Makes things easier to debug from calibration runs. If you want to be even more fancy, you could use TOPCAT to do a cross-match to see if anything else is in `GGSM.fits` that is missing from `pumav3` while you're at it.

**Comment by d3v-null** [comment]
Thanks Jack,
I crossmatched:
- GGSM (subset 5Jy `S_200>5` gives 1634 results)
- with `srclist_pumav3_EoR0LoBES_EoR1pietro_CenA-GP_2023-11-07.fits` , which I'll call JLB for short (subset 1Jy: `(NORM_COMP_PL>1 || NORM_COMP_CPL>1)` gives 1423 results )
- with max error 1amin I got 892 matches

<img width="2283" alt="image" src="https://github.com/user-attachments/assets/08bb2d87-6ad6-49b5-a140-c4212fe65b62">

Should I just chuck the remaining (1634-892=742) bright sources into JLB? Maybe it makes sense to add yet another suffix like `_GGSM5Jy`

**Comment by JLBLine** [comment]
Hmm looking at that plot shows this isn't super straight forward. The galactic plane stuff we have is based off of Jaiden's models, which have multiple components, so a simple 1 arcmin cutoff is likely going to get duplicates in there. Same for Cen A, there is a complicated model from Jaiden, which looks to be represented in the GGSM model as well, and is not being matched properly. You could go more conservative and do a 3 arcmin cutoff, but really you'll have to look at areas the two catalogues overlap and make sure there are no duplicates.

There is also the question of how deep you wanna go here, given GLEAM X is coming out at some point. We really should be replacing the whole catalogue based on that. Depends on how patient you want to be.
